Modules/Quotas/src/Quotas.Domain/Aggregates/Identities/Identity.cs

```diff
@@ -19,8 +22,11 @@ public Identity(string address, TierId tierId)
     public string Address { get; }
     public string TierId { get; }
 
-    public IndividualQuota CreateIndividualQuota(MetricKey metricKey, int max, QuotaPeriod period)
+    public Result<IndividualQuota, DomainError> CreateIndividualQuota(MetricKey metricKey, int max, QuotaPeriod period)
     {
+        if (max <= 0)
+            return Result.Failure<IndividualQuota, DomainError>(DomainErrors.TierQuotaMaxValueCannotBeLowerOrEqualToZero());
```

Comment:
Rename the error method (and the corresponding error code)
```suggestion
            return Result.Failure<IndividualQuota, DomainError>(DomainErrors.MaxValueCannotBeLowerOrEqualToZero());
```